meta/main.yml

```diff
@@ -1,6 +1,6 @@
 ---
 dependencies:
-  - role: elliotweiser.osx-command-line-tools
+  - src: 'elliotweiser.osx-command-line-tools'
```

Comment:
Awesome - thanks for the explanation - I'm new to ansible so this is very helpful to know!